Sources/Sentry/SentryProfiler.mm

```diff
@@ -437,17 +437,25 @@ + (SentryEnvelopeItem *)createProfilingEnvelopeItemForTransaction:(SentryTransac
         metrics[@"frozen_frame_renders"] = @{ @"unit" : @"nanosecond", @"values" : frozenFrames };
     }
 
-    const auto frameRates
-        = processFrameRates(_gCurrentFramesTracker.currentFrames.frameRateTimestamps, transaction);
-    if (frameRates.count > 0) {
-        metrics[@"screen_frame_rates"] = @{ @"unit" : @"hz", @"values" : frameRates };
+    if (slowFrames.count > 0 || frozenFrames.count > 0) {
```

Comment:
`m`: I would expect a change in a unit test when changing this logic. Is this covered?

Reply:
I don't think it is covered. I still have a todo list to add additional tests for the degenerate cases:
- no slow/frozen GPU timestamps in a profile
- there were slow/frozen frames, but fell outside of a transaction's start/end